### DISCUSSION TITLE
Missing block texture

### DISCUSSION THREAD

**Discussion by alazeprt** [FabricMC/fabric-api#2973] [Mod Dev Support] [answered]
When starting minecraft and loading my mod, the prompt "Using missing texture, file minecraft: textures/block/solar_panel_side. png not found" appears in the log. The texture of the block cannot be seen in the game, but I have added the texture of the block in the mod. Why?
Mod Link: https://github.com/alazeprt/SolarMod

**Comment by warjort** [comment, marked as answer]
Note the minecraft: namespace in the error
````minecraft:textures/block/solar_panel_side.png````

https://github.com/alazeprt/SolarMod/blob/5dd20199b25a95bd2cbd92c803940cbf35d675ea/src/main/resources/assets/solar/models/block/solar_panel.json#L6
````"side": "block/solar_panel_side"```` is in the minecraft namespace because you didn't specify a namepsace

You want it in the solar namespace to match where your texture is located:
````"side": "solar:block/solar_panel_side"````
